microshift_storage/understanding-persistent-storage-microshift.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="understanding-persistent-storage-microshift"]
+= Understanding persistent storage for MicroShift
+include::_attributes/attributes-microshift.adoc[]
+:context: understanding-persistent-storage-microshift
+
+toc::[]
+
+include::modules/microshift-storage-persistent-storage-overview.adoc[leveloffset=+1]
+
+include::modules/microshift-storage-persistent-storage-lifecycle.adoc[leveloffset=+1]
+
+include::modules/microshift-storage-persistent-storage-reclaim-manual.adoc[leveloffset=+2]
+
+include::modules/microshift-storage-persistent-storage-reclaim.adoc[leveloffset=+2]
+
+include::modules/microshift-storage-persistent-storage-pv.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+// xref needs completed
+* storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-rwx_persistent-storage-csi-vsphere[ReadWriteMany vSphere volume support]
```

Comment:
Not rendering the link.

Reply:
I don't think the ReadWriteMany vSphere page applies to MicroShift, right?